images/cluster-capacity/Dockerfile

```diff
@@ -0,0 +1,11 @@
+#
+# This is the cluster capacity tool.
+#
+# The standard name for this image is openshift/origin-cluster-capacity
+#
+FROM centos:centos7
+
+COPY bin/hypercc /bin/hypercc
```

Comment:
Would use same mechanism as #13642 -- can be a follow-up, depending on which lands first.